kombu/asynchronous/semaphore.py

```diff
@@ -28,13 +37,18 @@ class LaxBoundedSemaphore:
         HELLO 3
     """
 
-    def __init__(self, value):
+    def __init__(self, value: int) -> None:
         self.initial_value = self.value = value
-        self._waiting = deque()
+        self._waiting: Deque[Tuple[P, P.args, P.kwargs]] = deque()
```

Comment:
I don't think this is a possible use case, because the second argument is actually a list, and the third is actually a dictionary, also ParamSpec is only used with a callable. This passes on mypy? In any case, it should be at least:
```suggestion
        self._waiting: Deque[Tuple[Callable[P, None], P.args, P.kwargs]] = deque()
```

Reply:
Mypy did not like either, so I reverted to `Deque[tuple]` for now.